### DISCUSSION TITLE
让启动器缩小到角标  用更小的显示面积干更多的事

### DISCUSSION THREAD

**Discussion by NanGauCover** [Meloong-Git/PCL#1133] [提问] [answered]
启动器缩小到角标按钮：
![KI~CQU YBSH`~@ 97CIV _O](https://user-images.githubusercontent.com/96673414/216557101-b7371aa3-8db9-47ff-8e0c-b9bc6144f639.png)
启动器缩小到任务栏效果（如果开着游戏将会在下面显示红点）：
![QY(7`Y@X46USXERKX{38_TC](https://user-images.githubusercontent.com/96673414/216557353-d42059e9-1422-49d1-8eca-3c6b8c5b4f87.png)
右键角标所呈现的效果（编辑角标右键窗口需要内测用户）：
![QY(7`Y@X46USXERKX{38_TC - 副本](https://user-images.githubusercontent.com/96673414/216557578-60115f70-6625-4103-a700-0275243307e1.png)

**Comment by NanGauCover** [comment]
漏了一张图
![KI~CQU YBSH`~@ 97CIV _O](https://user-images.githubusercontent.com/96673414/216557946-16377ab2-d587-4247-874a-fd639fe3efd6.png)

**Comment by WForst-Breeze** [comment, marked as answer]
#772 提及相似建议，此建议是其扩展和补充

如有新功能建议，请提交 [新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml) ，感谢您的支持

**Comment by MoYuan-CN** [comment]
进 #1162 了 去顶这条
